api/controller.py

```diff
@@ -267,6 +300,10 @@ def setup_adobe_vendor_id(self, library):
             vendor_id = adobe.username
             node_value = adobe.password
             if vendor_id and node_value:
+                if self.adobe_vendor_id:
+                    self.log.warn(
+                        "Multiple libraries define an Adobe Vendor ID integration. This is not supported and the last library seen will take precedence."
+                    )
```

Comment:
I'm not making a big deal out of this because the Adobe Vendor ID integration is going away in the medium term.